login-workflow/docs/services.md

```diff
@@ -51,15 +51,18 @@ constructor(pxbAuthConfig: PxbAuthConfig) {
     -   Custom email regex requirements.
     -   Default: `undefined`
 -   **customPasswordRequirements** (optional): _`PasswordRequirement[]`_
-    -   Array of custom password requirements.
+    -   Array of custom password requirements. 
     -   Default: `[]`
+-   **customTranslations** (optional): _`PxbAuthTranslations`_
```

Comment:
Can we add some documentation around this? What happens if a user supplies a custom translation file with missing fields? Should we link them (or point them) to information about what strings they will have to include?

Reply:
We have a type `PxbAuthTranslations` for them to use; as long as they use that type then they can't accidently forget fields.

Reply:
I can add a blurb saying to look at the other translation files for reference.